.gitmodules

```diff
@@ -118,3 +118,9 @@
 [submodule "src/code.cloudfoundry.org/go-pubsub"]
 	path = src/code.cloudfoundry.org/go-pubsub
 	url = https://github.com/cloudfoundry-incubator/go-pubsub
+[submodule "src/code.cloudfoundry.org/grpc-throughputlb"]
+	path = src/code.cloudfoundry.org/grpc-throughputlb
+	url = https://github.com/cloudfoundry-incubator/grpc-throughputlb
+[submodule "src/github.com/alecthomas/gometalinter"]
```

Comment:
I propose leaving tooling-related concerns out of our submodules, and instead add a step to install any tools to our test script.

Reply:
That got pulled in in a conflict during rebase, and thought it was added by loggregator. I will remove.